src/CommandingOfficers/MeteorParams.java

```diff
@@ -0,0 +1,116 @@
+package CommandingOfficers;
+
+import java.util.ArrayList;
+
+import Engine.Utils;
+import Engine.XYCoord;
+import Engine.GameEvents.MassDamageEvent;
+import Terrain.GameMap;
+import Terrain.MapMaster;
+import Units.Unit;
+
+public class MeteorParams
+{
+  public XYCoord target; // This is allowed to be null. If it is, then this class will produce a no-op MassDamageEvent.
+  public int power = 3; // Silo
+  public int radius = 2; // ditto
```

Comment:
Silo?

Reply:
Rachel and VB's damage matches Silo damage?

Reply:
Oh. I guess I probably should have made the connection, but for once, I'm not sure we need a comment to explain this. I guess it doesn't hurt though. Unless the reader just gets confused.